src/components/themed/ControlPanel.tsx

```diff
@@ -42,7 +43,7 @@ import { TitleText } from '../text/TitleText'
 import { DividerLine } from './DividerLine'
 
 interface Props {
-  navigation: NavigationProp<'controlPanel'>
+  navigation: NavigationBase
```

Comment:
This is not a scene component, but it might make sense to be more strict with the type here. Keep it as it was?